### DISCUSSION TITLE
CancelledError after quit

### DISCUSSION THREAD

**Discussion by PyPandaPo** [fastapi/fastapi#8553] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI

app = FastAPI()


@app.get("/")
async def root():
    return {"message": "Hello World"}
```

### Description

I ran a example code from the documentation and if I terminate the program (Press CTRL+C), the program crashes with an error.

```
INFO:     Will watch for changes in these directories: ['/Users/vitaly/Dev/pet-projects/todolist']
INFO:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
INFO:     Started reloader process [4558] using watchgod
INFO:     Started server process [4560]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
^CINFO:     Shutting down
INFO:     Finished server process [4560]
ERROR:    Traceback (most recent call last):
  File "/Users/vitaly/Library/Caches/pypoetry/virtualenvs/todolist-UnSYqD3H-py3.10/lib/python3.10/site-packages/starlette/routing.py", line 624, in lifespan
    await receive()
  File "/Users/vitaly/Library/Caches/pypoetry/virtualenvs/todolist-UnSYqD3H-py3.10/lib/python3.10/site-packages/uvicorn/lifespan/on.py", line 135, in receive
    return await self.receive_queue.get()
  File "/opt/homebrew/Cellar/python@3.10/3.10.1/Frameworks/Python.framework/Versions/3.10/lib/python3.10/asyncio/queues.py", line 159, in get
    await getter
asyncio.exceptions.CancelledError

INFO:     Stopping reloader process [4558]
```

What could be the problem and how to fix it?

### Operating System

Linux, macOS

### FastAPI Version

0.71.0

### Python Version

3.10.1

**Comment by ravivaghasiya1998** [comment]
hello
is there any solution for this error ?
